Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -596,6 +589,38 @@
   SysCallLib|MmSupervisorPkg/Library/SysCallLib/SysCallLib.inf
   CpuLib|MmSupervisorPkg/Library/BaseCpuLibSysCall/BaseCpuLib.inf
 
+#########################################
+# Advanced Logger Libraries
+#########################################
+[LibraryClasses]
+  DebugLib|AdvLoggerPkg/Library/BaseDebugLibAdvancedLogger/BaseDebugLibAdvancedLogger.inf
+  AdvancedLoggerHdwPortLib|AdvLoggerPkg/Library/AdvancedLoggerHdwPortLib/AdvancedLoggerHdwPortLib.inf
+  AssertLib|AdvLoggerPkg/Library/AssertLib/AssertLib.inf
+  AdvancedLoggerAccessLib |AdvLoggerPkg/Library/AdvancedLoggerAccessLib/AdvancedLoggerAccessLib.inf
+
+[LibraryClasses.common.MM_CORE_STANDALONE, LibraryClasses.common.MM_STANDALONE, LibraryClasses.common.PEIM, LibraryClasses.common.PEI_CORE]

```

Comment:
There is MM_CORE_STANDALONE instances for X86 arch. Have you considered enabling that?

Reply:
Ah that wasn't in the Advanced Logger doc. It's currently hanging when copying the HOB with those library instances. I'll dig into it more.

Reply:
Looks like the attempt to fetch the advanced logger info via gAdvancedLoggerHobGuid fails and the assert in MM_CORE is not being properly routed resulting in a CpuPause. I suspect this might be what's happening when I try to use the PEI_CORE advanced logger as well. Looking through all references to gAdvancedLoggerHobGuid I can't tell what entity is supposed to create the HOB. Do you know?

Reply:
Oh, it would expect the gAdvancedLoggerHobGuid to be available by the time MM_CORE is launched. Do you mean that PEI phase did not launch advanced logger? We probably should handle that first.

Reply:
I see now the PEI_CORE AdvancedLoggerGetLoggerInfo creates the HOB entry. I'll continue working backwards through the phases.

Reply:
I've tried several ways to tease out what's causing the hang but no luck. Are you find with just enabling DXE advanced logger for now?

Reply:
Note that the hang that's hard to pinpoint is PEI_CORE before the debugger is useful

Reply:
I think just DXE is fine for now. Please also log an issue to track the remaining work on the advanced logger topic.